test/modellib/CategoryMapperModel.cpp

```diff
@@ -61,7 +61,7 @@ bool CategoryMapperLibBuilder<T1, T2>::prepareInputs() {
   int64_t shape[1] = {static_cast<int64_t>(input.size())};
   list[0] = createOMTensor<T1>(input, shape, 1,
       (std::is_same<T1, int64_t>::value) ? ONNX_TYPE_INT64 : ONNX_TYPE_STRING);
-  inputs = omTensorListCreateWithOwnership(list, 1, true);
+  inputs = omTensorListCreate(list, 1);
```

Comment:
I believe we have now a memory leak. In the old code, the omTensorListCreateWithOwnership copied the list and would delete it on destruction of the list.

In the new code, tensor list create alloc and copy. When destroying, it will delete its copy. But in the above case, no-one deletes the code on line 56. 

Since dw don't need ownership anymore, I believe we can just declare a list on the stack:
```
OMTensor* list[1];
```
which will be naturally freed when the function returns.

Please check for similar leaks elsewhere, and ask @cjvolzka to possibly run valgrind to double check we did not introduce new leaks.

Reply:
@AlexandreEichenberger That was already on my todo list ;)
I'm working on a build now to try a few models.

Reply:
Thanks for the feedback @AlexandreEichenberger .... will make the needed changes. I appreciate the help

Reply:
@cjvolzka I just realized....that is `omTensorCreateWithOwnership`. This PR only updated `omTensorCreateListWithOwnership`. So I kept that as is, are we planning to change `omTensor` as well ?